README.md

```diff
@@ -242,3 +242,71 @@ Testing looks something like this:
 
 But in production use you'd expect it more like
    `curl <auth to your proxy> https://ssoproxy.example.com/enroll_user -d @~/.ssh/bob.pub`
+
+### Github Support
+
+Sharkey supports issuing certificates for SSH access to Github by mapping SAML identities to Github usernames through 
+the following GraphQL query:
+```
+query {
+  organization(login: "squareup") {
+    samlIdentityProvider {
+      ssoUrl
+      externalIdentities(first: 100, after: null) {
+        edges {
+          node {
+            guid
+            samlIdentity {
+              nameId
+            }
+            user {
+              login
+            }
+          }
+        }
+        pageInfo {
+          hasNextPage
+          endCursor
+        }
+      }
+    }
+  }
+}
+```
+
+In order to enable support for Github, add the following stanza to Sharkey's server config:
+```
+github:
```

Comment:
let's keep this together with the example config section (for the server) and comment on each line